packages/core/src/index.ts

```diff
@@ -858,6 +859,10 @@ class Hopr extends EventEmitter {
     return await ethereum.getPublicKeyOf(addr)
   }
 
+  public async signMessage(message: Uint8Array) {
+    return await this.id.privKey.sign(u8aConcat(new TextEncoder().encode('HOPR Signed Message: '), message))
```

Comment:
Since the prefix is security critical, please document the reasoning here.

Reply:
Good point, updating.

Reply:
Added NB.